src/Logo/index.ts

```diff
@@ -1,4 +0,0 @@
-import Logo from "./Logo";
-import BetaTag from "./BetaTag";
-export default Logo;
-export { BetaTag };
```

Comment:
I kind of like the `index.ts` file imports the other named files! Don't want to start a holy war here, just sayin' :P

Reply:
Womp